### PR TITLE
feat(STN-155):  3 column layout and styling refinement

### PR DESCRIPTION
**Pull request by silverli**
# READY FOR REVIEW

## Summary
**[JIRA CARD 155](https://sparkbox.atlassian.net/browse/STN-155)**

Adds template and refined styles for 3 column layout, and 3 column with image styling tweaks.

See screenshots in a comment below for visual design with background colors for extra clarity.

## Steps to Test
1. Use layout builder to add 3 or 4 layouts using 3 column layout. Add one with only main content, one with 1 sidebar, and 1 with 2 sidebars.
2. Inside `humsci_basic` run `npm run test` and make sure all tests pass.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)

### CODE REVIEW

**Comment by silverli** [comment]
## 2 sidebars
2-sidebar-lg
2-sidebar-md
2-sidebar-sm

## 1 sidebar
1-sidebar-left-lg
1-sidebar-left-md
1-sidebar-right-lg
1-sidebar-right-md
1-sidebar-right-sm


## no sidebar
no-sidebar-md
nosidebar-lg

File: docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.general.scss [hunk 1]
```diff
@@ -1,13 +1,18 @@
 .hb-page-width {
-  @include hb-page-width;
```
Comment (silverli) [inline docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.general.scss:2]: I dont think this mixin makes sense for our styles.. what do others think? I think Merani had wanted 3rem to kinda pad everything in our design

File: docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.three_column_layouts.scss [hunk 2]
```diff
@@ -0,0 +1,159 @@
+$hb-sidebar-width: 16.5rem;
```
Comment (silverli) [inline docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.three_column_layouts.scss]: sorry for renaming this.. make sure to compare to the original file `three_column_with_image.scss`

File: docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.general.scss (hunk 1 above)
Comment (kaseybon) [reply docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.general.scss:2]: I think the goal was to use the Decanter mixin but if that isn't going to work for our layouts we can abandon it.

File: docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.general.scss [hunk 3]
```diff
@@ -1,13 +1,18 @@
...
+  max-width: 1500px;
+  margin: 0 auto;
 }
 
 // Other one-off applications of @include hb-page-width.
 // If a component needs *any* additional styles, move the rule into /src/scss/components.
 .layout-builder__message {
-  @include hb-page-width;
+  // @include hb-page-width;
```
Comment (kaseybon) [inline docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.general.scss]: These comments should be removed.

File: docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.general.scss [hunk 4]
```diff
@@ -1,13 +1,18 @@
...
+  max-width: 1500px;
```
Comment (kaseybon) [inline docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.general.scss]: Let's make a variable for this.

File: docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.three_column_layouts.scss (hunk 2 above)
Comment (kaseybon) [reply docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.three_column_layouts.scss]: Renaming is fine but we should move variable declarations to `/settings/variables.general.scss`.

File: docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.three_column_layouts.scss [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+
+.hb-three-column-w-image,
+.hb-three-column {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 1rem 3rem;
```
Comment (kaseybon) [inline docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.three_column_layouts.scss]: Add any margins as variables in `/settings/variables.general.scss`. If we do need them to behave differently at different breakpoints then we can make a scss map.

File: docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.three_column_layouts.scss [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+    }
+
+    @include grid-media('lg') {
+      flex: 1 1 100%;
+    }
+
+    .hb-three-column-w-image--one-sidebar &,
+    .hb-three-column--one-sidebar & {
+      @include grid-media('2xl') {
+        max-width: 68.75rem;
```
Comment (kaseybon) [inline docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.three_column_layouts.scss]: More variables plz

File: docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.three_column_layouts.scss [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+      }
+    }
+  }
+
+  &__main-top {
+    // should this style apply to the 3 column grid?
+    // need to make sure I didnt mess up node news details page
```
Comment (kaseybon) [inline docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.three_column_layouts.scss]: Yeah, we're affecting the three col w/ image. I would think we would want those layouts to match for the most part so maybe pair with Merani on that?

**Review by kaseybon — CHANGES_REQUESTED** [review]

File: docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.three_column_layouts.scss (hunk 7 above)
Comment (silverli) [reply docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.three_column_layouts.scss]: Yeah we'll keep the styles separate for now

**Comment by silverli** [comment]
@kaseybon I'll still need to commit the built css when this is ready

**Review by kaseybon — APPROVED** [review]

**Pull request merged**